includes/Ad_Manager.php

```diff
@@ -77,15 +77,21 @@ public function print_ad_manager_tag() {
 		if ( ! $enabled || ! $slot ) {
 			return;
 		}
+
+		$application_json = [];
+
+		$ad_attributes = [];
+		$ad_attributes[ 'type' ] = 'doubleclick';
+		$ad_attributes[ 'data-slot' ] = $slot;
+
+		$application_json['ad-attributes'] = $ad_attributes;
+		
+		$application_json = apply_filters( 'web_stories_ad_manager_json', $application_json );
+
 		?>
 		<amp-story-auto-ads>
 			<script type="application/json">
-				{
-					"ad-attributes": {
-						"type": "doubleclick",
-						"data-slot": "<?php echo esc_js( $slot ); ?>"
-					}
-				}
+				<?php echo wp_json_encode( $application_json, JSON_UNESCAPED_SLASHES|JSON_PRETTY_PRINT ); ?>
```

Comment:
I don't think we need to pretty print.

Is there a good reason for unescaped slashes?

Elsewhere we just use `JSON_UNESCAPED_UNICODE` as the only flag.

Reply:
i need to use JSON_UNESCAPED_SLASHES because the id slot will be escaped here is an example of the output:
{"ad-attributes":{"type":"doubleclick","data-slot":"\/123456789\/a4a\/amp_story_dfp_example"}}